### PR TITLE
Removed duplicate button to link delivery service to server

### LINKED ISSUE #5216: TP has "Link delivery services to server" button as well as under the "More" menu
## I'm submitting a ...
-  bug report

## Traffic Control components affected ...
-  Traffic Portal

## Current behavior:
TP has two buttons that both link delivery services to a server in the `/#!/servers/id/delivery-services` page:
image

## Expected behavior:
The `More` menu item for linking delivery services to the server should be removed.

## Minimal reproduction of the problem with instructions:
In your TP, go to `/#!/servers/<your-server-ID>/delivery-services`.

### PR DESCRIPTION
**Pull request by rimashah25**
## What does this PR (Pull Request) do?

- [x] This PR fixes #5216 

## Which Traffic Control components are affected by this PR?

- Traffic Portal

## What is the best way to verify this PR?

Login to TP UI.
Go to https://<TP'surl:port>/#!/servers/<server_id>/delivery-services
Should see `More` button only if conditions are met (server == edge and DS.length > 0).
image

## If this is a bug fix, what versions of Traffic Control are affected?

- master (4cd6fba)
- 4.1.0

## The following criteria are ALL met by this PR

- [x] I have explained why tests are unnecessary
- [x] I have explained why documentation is unnecessary
- [x] This PR includes an update to CHANGELOG.md OR such an update is not necessary
- [x] This PR includes any and all required license headers
- [x] This PR ensures that database migration sequence is correct OR this PR does not include a database migration
- [x] This PR **DOES NOT FIX A SERIOUS SECURITY VULNERABILITY** (see [the Apache Software Foundation's security guidelines](https://www.apache.org/security/) for details)

## Additional Information

The documentation is upto date. No test required since logic hasn't changed.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -75,11 +75,12 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
...
 
 ### Fixed
-- Fixed #5188 - DSR (delivery service request) incorrectly marked as complete and error message not displaying when DSR fulfilled and DS update fails in Traffic Portal. [Related Github issues](https://github.com/apache/trafficcontrol/issues/5188)
-- Fixed #3455 - Alphabetically sorting CDN Read API call [Related Github issues](https://github.com/apache/trafficcontrol/issues/3455)
-- Fixed #5010 - Fixed Reference urls for Cache Config on Delivery service pages (HTTP, DNS) in Traffic Portal. [Related Github issues](https://github.com/apache/trafficcontrol/issues/5010)
-- Fixed #5147 - GET /servers?dsId={id} should only return mid servers (in addition to edge servers) for the cdn of the delivery service if the mid tier is employed. [Related github issues](https://github.com/apache/trafficcontrol/issues/5147)
-- Fixed #4981 - Cannot create routing regular expression with a blank pattern param in Delivery Service [Related github issues](https://github.com/apache/trafficcontrol/issues/4981)
+- Fixed # - Removed duplicate button to link delivery service to server [Related Github issue](https://github.com/apache/trafficcontrol/issues/5216)
```
Comment (mitchell852) [inline CHANGELOG.md:78]: can u just move this one line under the unreleased/fixed section?

Comment (mitchell852) [reply CHANGELOG.md:78]: oh and add the github number

Comment (mitchell852) [reply CHANGELOG.md:78]: also, not sure why it's showing so many more changes in the changelog...

Comment (rimashah25) [reply CHANGELOG.md]: Sure and Done.

**Review by mitchell852 — APPROVED** [review]
works like a charm

**Pull request merged**